frontend/tet/admin/src/components/jobPostings/JobPostings.tsx

```diff
@@ -22,20 +22,21 @@ const JobPostings: React.FC = () => {
 
   console.dir(data);
 
-  const content = data?.length ? (
-    <JobPostingsList postings={data} />
-  ) : (
-    <>
-      <p>{t('common:application.jobPostings.noPostingsFound')}</p>
-      <Button onClick={() => router.push('/new')}>{t('common:application.jobPostings.addNewPosting')}</Button>
-    </>
-  );
+  const content =
+    data?.draft && data?.published ? (
+      <JobPostingsList draft={data.draft} published={data.published} />
+    ) : (
+      <>
+        <p>{t('common:application.jobPostings.noPostingsFound')}</p>
```

Comment:
This is never shown. If `data.draft` and `data.published` are both `[]` `<JobPostingsList>` is still rendered.